src/sentry/tasks/derive_code_mappings.py

```diff
@@ -90,13 +90,17 @@ def derive_code_mappings(
             logger.warning("The org has uninstalled the Sentry App.", extra=extra)
             return
 
-        raise error  # Let's be report the issue
+        raise error  # Let's report the issue
     except UnableToAcquireLock as error:
         extra["error"] = error
         logger.warning("derive_code_mappings.getting_lock_failed", extra=extra)
         # This will cause the auto-retry logic to try again
         raise error
 
+    if not trees:
+        logger.error("The tree is empty. Investigate.")
```

Comment:
I'm curious about the goal for these log messages where we call out 'pending investigation' or 'investigate'. Is that to retain context about the feature? How frequently do folks comb through the logs looking for these types of messages?